ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-ref.yaml

```diff
@@ -36,6 +36,13 @@ ref:
   - name: OUTBOUND_TYPE
     default:  ""
     documentation: |-
-      This value represents the routing strategy used to connect the cluster to the internet 
+      This value represents the routing strategy used to connect the cluster to the internet
+  - name: PUBLISH
```

Comment:
Let us read `publish` from install-config.yaml instead of getting it from ENV, just like https://github.com/openshift/release/pull/30855/files#diff-56bc3e48dda0b0233a4d83f96ae56773ff74ecd6ed86e316b880d041a92af2e2R26